upgrade

```diff
@@ -30,82 +28,110 @@ if [ "$(whoami)" != "root" ]; then
+  sudo apt-get dist-upgrade -y
+  sudo do-release-upgrade -f DistUpgradeViewNonInteractive
+
+  export NPM_BIN=$(npm -g bin)
+
+  decho "stopping lamassu-server"
+  supervisorctl stop lamassu-server >> ${LOG_FILE} 2>&1
+  supervisorctl stop lamassu-admin-server >> ${LOG_FILE} 2>&1
+
+  decho "unlinking ${NPM_BIN}/lamassu* old executables"
+  find ${NPM_BIN} -type l \( -name "lamassu-*" -or -name "hkdf" \) -exec rm -fv {} \; >> ${LOG_FILE} 2>&1
+
+  if [ -d "/usr/lib/node_modules/lamassu-server" ]; then
+      BKP_NAME=lamassu-server-$(date +%s)
+      decho "renaming old lamassu-server instance to ${BKP_NAME}"
+      mv -v "/usr/lib/node_modules/lamassu-server" "/usr/lib/node_modules/${BKP_NAME}" >> ${LOG_FILE} 2>&1
+  fi
+
+  decho "updating lamassu-server to Electric Enlil v7.5.0-beta.0..."
+  npm -g install lamassu/lamassu-server#v7.5.0-beta.0 --unsafe-perm >> ${LOG_FILE} 2>&1
+
+  decho "rebuilding npm deps"
+  cd $(npm root -g)/lamassu-server/ >> ${LOG_FILE} 2>&1
+  npm rebuild >> ${LOG_FILE} 2>&1
+
+  {
+  decho "running migration"
+    lamassu-migrate >> ${LOG_FILE} 2>&1
+  } || { echo "Failure running migrations" ; exit 1 ; }
+
+  lamassu-migrate-config >> ${LOG_FILE} 2>&1
+
+  decho "update to mnemonic"
+  lamassu-update-to-mnemonic --prod >> ${LOG_FILE} 2>&1
+
+  decho "update ofac sources"
+  lamassu-ofac-update-sources >> ${LOG_FILE} 2>&1
+
+  decho "updating supervisor conf"
+  perl -i -pe 's/command=.*/command=$ENV{NPM_BIN}\/lamassu-server/g' /etc/supervisor/conf.d/lamassu-server.conf >> ${LOG_FILE} 2>&1
+  perl -i -pe 's/command=.*/command=$ENV{NPM_BIN}\/lamassu-admin-server/g' /etc/supervisor/conf.d/lamassu-admin-server.conf >> ${LOG_FILE} 2>&1
+  cat <<EOF >> /etc/supervisor/supervisord.conf
+[inet_http_server]
+port = 127.0.0.1:9001
+EOF
+
+  decho "updating lamassu-server"
+  supervisorctl update lamassu-server >> ${LOG_FILE} 2>&1
+  supervisorctl update lamassu-admin-server >> ${LOG_FILE} 2>&1
+  supervisorctl start lamassu-server >> ${LOG_FILE} 2>&1
+  supervisorctl start lamassu-admin-server >> ${LOG_FILE} 2>&1
+  
+  decho "updating backups conf"
+  BACKUP_CMD=${NPM_BIN}/lamassu-backup-pg
+  BACKUP_CRON="@daily $BACKUP_CMD > /dev/null"
+  ( (crontab -l 2>/dev/null || echo -n "") | grep -v '@daily.*lamassu-backup-pg'; echo $BACKUP_CRON ) | crontab - >> $LOG_FILE 2>&1
+  $BACKUP_CMD >> $LOG_FILE 2>&1
+
+  decho "updating motd scripts"
+  set +e
+  chmod -x /etc/update-motd.d/*-release-upgrade 2>/dev/null
+  chmod -x /etc/update-motd.d/*-updates-available 2>/dev/null
+  chmod -x /etc/update-motd.d/*-reboot-required 2>/dev/null
+  chmod -x /etc/update-motd.d/*-help-text 2>/dev/null
+  chmod -x /etc/update-motd.d/*-cloudguest 2>/dev/null
+  chmod -x /etc/update-motd.d/*-motd-news 2>/dev/null
+  set -e
+
+  # reset terminal to link new executables
+  hash -r
+
+  rm -r /var/lock/lamassu-update
+
+  echo "Update part 3 complete! Restart the server to complete the process."
+elif [[ "$UBUNTU_VERSION" == "20.04" ]]; then
```

Comment:
This doesn't mean the system is up-to-date. This same script needs to accomodate operators who had installed v7.5.0 fresh for the first time, starting with 20.04, and who in the future need to go from v7.5.0 to v7.5.1, or whatever the latest version is.